### PR TITLE
Add components to documentation site

### PR DESCRIPTION
**Pull request by gadgetfather**
I have added created components for the documentation site. Please review them. I have made all P0 and some of the P1 components
Link : https://mana-ui.netlify.app/
image
image
image

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
Great and clean code!

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="/style.css">
```
Comment (shaik-saad) [inline index.html:7]: You can add a dot before the forward slash

File: Documentation/utilities-page.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="#" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/utilities-page.html]: button has a href here, you can remove it.

File: Components/list-component/list.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
+@import url(/common.css);
+
+.list{
+    border: 1px solid black;
+    width: 12rem;
+    display: flex;
+    flex-direction: column;
+    gap: var(--spacing7);
+    /* align-items: center; */
```
Comment (shaik-saad) [inline Components/list-component/list.css]: if not needed, you can remove this comment.

File: Components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+}
+/* vertical card */
+.card-vertical{
+    width: 14.5rem;
+}
+
+/* horizontal card */
+.card-horizontal{
+ width: 30rem;
+ flex-direction: row;
```
Comment (shaik-saad) [inline Components/card/card.css:77]: flex is by default row, you can remove this.

File: Components/list-component/list.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    padding: 0.3rem 0.2rem;
+    background-color: lightgray;
+}
+
+/* spaced */
+.list-spaced{
+    flex-direction: row;
+    align-items: center;
+    width: 30rem;
+    /* list-style: circle; */
```
Comment (shaik-saad) [inline Components/list-component/list.css]: if not needed you can remove this one.

File: Components/list-component/list.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+.list-stacked li{
+    border: 1px solid black;
+    width: 100%;
+    padding: 0.3rem 0.2rem;
+    background-color: lightgray;
+}
+
+/* spaced */
+.list-spaced{
+    flex-direction: row;
```
Comment (shaik-saad) [inline Components/list-component/list.css:32]: flex-direction: is by default row, so you can remove this line

File: Documentation/alert-page.html [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/alert-page.html]: Button has a href attribute here

File: Documentation/badge-page.html [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/badge-page.html]: href attribute here too.

File: Documentation/button-page.html [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/button-page.html]: href attribute here too.

File: Documentation/card-page.html [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+            <a href="/Documentation/button-page.html" class="btn btn-text-primary">Button</a>
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/card-page.html]: href attribute here too

File: Documentation/color-page.html [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/color-page.html]: href here too

File: Documentation/image-page.html [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/image-page.html]: you can remove this href

File: Documentation/index.html [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/index.html]: href here too

File: Documentation/input-page.html [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/input-page.html]: button has a href

File: Documentation/list-page.html [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive sidebar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/list-page.html]: href

File: Documentation/modal-page.html [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+            <a href="/Documentation/card-page.html" class="btn btn-text-primary">Card</a>
+            <a href="/Documentation/image-page.html" class="btn btn-text-primary">Image</a>
+            <a href="/Documentation/input-page.html" class="btn btn-text-primary">Input</a>
+            <a href="/Documentation/list-page.html" class="btn btn-text-primary">List</a>
+            <a href="/Documentation/modal-page.html" class="btn btn-text-primary">Modal</a>
+        </aside>
+        <div class="main-content">
+            <!-- responsive side bar -->
+            <div class="responsive-sidebar-container">
+                <button class="btn btn-secondary hamburger-menu" href="#"><i class="fas fa-bars"></i></button>
```
Comment (shaik-saad) [inline Documentation/modal-page.html]: href here too

File: Documentation/style.css [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+    "nav nav nav nav"
+    "sidebar page page ..."
+    "... page page ...";
+}
+/* navbar */
+.navbar{
+    grid-area: nav;
+    display: flex;
+    align-items: center;
+    /* border: 2px solid red; */
```
Comment (shaik-saad) [inline Documentation/style.css:26]: if not needed, you can remove this comment

File: Documentation/style.css [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+}
+
+/* sidebar */
+.sidebar{
+    grid-area: sidebar;
+    display: flex;
+    padding-left: 1.1rem;
+    flex-direction: column;
+    gap: var(--spacing6);
+    /* border: 1px solid gold; */
```
Comment (shaik-saad) [inline Documentation/style.css:61]: if not needed, you can remove this comment

File: Documentation/style.css [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+
+/* main-content */
+.main-content{
+    grid-area: page;
+    margin-left: 2rem;
+    /* position: relative; */
+    /* border: 1px solid green; */
+    padding: 0 1.2rem;
+    /* background-color: var(--primary-color); */
+}
```
Comment (shaik-saad) [inline Documentation/style.css:77]: if these lines are not needed, you can remove them

File: Documentation/style.css [hunk 20]
```diff
@@ -0,0 +1,224 @@
...
+    display: flex;
+    flex-direction: column;
+    padding: var(--spacing8);
+    background-color: var(--secondary-hover-color);
+    align-items: center;
+    margin:1rem 3rem    ;
+}
+
+.component-horizontal{
+    flex-direction: row;
```
Comment (shaik-saad) [inline Documentation/style.css:133]: it's by default row, so you can remove this line

**Review by shaik-saad — CHANGES_REQUESTED** [review]
Added some suggestions, do check them out

File: Components/card/card.css (hunk 4 above)
Comment (gadgetfather) [reply Components/card/card.css:77]: No actually for card container, i had made it vertical, so to specifically make it horizontal i  gave flex direction row to

**Review by VishalPatil18 — APPROVED** [review]
- The installation code embed width is broken. You could fix that.
Screenshot 2022-02-03 at 23 17 08

- The close buttons on the alerts could have a cursor of pointer when they're being hovered upon.

Screenshot 2022-02-03 at 23 18 56

The rest looks good. Great stuff @gadgetfather

**Review by shaik-saad — APPROVED** [review]
Looks good Akshay! you can go ahead with merge.

**Pull request merged**